src/components/ConnectorName.tsx

```diff
@@ -1,34 +1,21 @@
 import { Stack } from '@mui/material';
 import ConnectorIcon from 'components/ConnectorIcon';
-import { OpenGraph } from 'types';
 import useConstant from 'use-constant';
-import { getConnectorIcon } from 'utils/misc-utils';
 
 interface Props {
-    iconPath?: OpenGraph | string | null;
+    iconPath?: string | null;
     iconSize?: number;
-    connector: OpenGraph | string | null;
+    title: string | null;
 }
 
-function ConnectorName({ connector, iconPath, iconSize }: Props) {
-    const connectorName = useConstant(() =>
-        typeof connector === 'string'
-            ? connector
-            : connector === null
-            ? ''
-            : connector.title
-    );
```

Comment:
These changes are fine. They were here as the original way to allow the connector name to default to the image. If we only allow a string to be passed in that is fine.